src/extension/gogoduck/README.md

```diff
@@ -0,0 +1,76 @@
+# GoGoDuck
+
+GoGoDuck is a simple NetCDF file aggregator, using nco tools (ncks, ncrcat, ncatted, etc).
+
+## Requirements
+
+ * nco tools, a recent version! (>= 4.3.4)
+ * java/maven
+
+## Usage:
+
+To build:
+```
+$ mvn clean install
+```
```

Comment:
I get [build errors](https://gist.github.com/jkburges/76171129d9e6101bccf9) when doing this.